auto3dseg/algorithm_templates/dints/configs/transforms_train.yaml

```diff
@@ -3,7 +3,7 @@ image_key: image
 label_key: label
 transforms_train:
   _target_: Compose
-  lazy_evaluation: "$@training#transforms#lazy_resampling"
+  lazy_evaluation: "$@transforms#lazy_resampling"
```

Comment:
This also needs to be reverted to "$@training#transforms#lazy_resampling"

Reply:
I have updated this key.